src/encoded/static/components/medicationChart.js

```diff
@@ -124,7 +124,7 @@ class MedicationChart extends React.Component {
       },
       textposition: 'left',
       showlegend: false,
-      hovertemplate: 'Diagnosis date: ' + maxDate + '<extra></extra>',
+      hovertemplate: 'Deceased date: ' + maxDate + '<extra></extra>',

```

Comment:
Why did this change?

Reply:
I don't know how you merged previous branch, just found this bug..

Reply:
Ya weird bug. Everything else looks good